samples/drivers/spi_flash_at45/sample.yaml

```diff
@@ -24,3 +24,22 @@ tests:
         - "Checking the read content... OK"
         - "Putting the flash device into low power state... OK"
```

Comment:
This line should be removed from here, as without `CONFIG_DEVICE_POWER_MANAGEMENT` enabled it will not appear. Similarly the line `- "Using \\w+...`. It won't be present in configurations without `CONFIG_FLASH_PAGE_LAYOUT` enabled.

Reply:
In fact, when I submit the PR I was not expecting the test to pass. I opened #28099 because of relation from device PM with CONFIG_SYS_POWER_MANAGEMENT. This is another motive to create the overlay-pm.conf.